pkg/grapicmd/di/providers.go

```diff
@@ -25,9 +26,10 @@ func ProvideScriptLoader(ctx *grapicmd.Ctx, executor excmd.Executor) module.Scri
 	return script.NewLoader(ctx.FS, executor, ctx.RootDir.String())
 }
 
-func ProvideInitializeProjectUsecase(ctx *grapicmd.Ctx, gexCfg *gex.Config, ui cli.UI, generator module.Generator, excmd excmd.Executor) usecase.InitializeProjectUsecase {
+func ProvideInitializeProjectUsecase(ctx *grapicmd.Ctx, gexCfg *gex.Config, ui cli.UI, fs afero.Fs, generator module.Generator, excmd excmd.Executor) usecase.InitializeProjectUsecase {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
line is 184 characters (lll)